webview/shared/Editor/index.tsx

```diff
@@ -20,6 +20,7 @@ const Editor = (props) => {
         minimap: { enabled: false },
         scrollBeyondLastLine: false,
         theme: "vs-dark",
+        automaticLayout: true,
```

Comment:
Shifting to the **GraphQL** Tab (under Body) causes the editor to keep expanding horizontally. Have a hunch that this is being caused by `automaticLayout`. Could you fix this?

Reply:
automaticLayout is set to true inorder to make the editor expand when resizing, if set to false, when resizing it wouldn't take the new area.
I am working on this bug.

Reply:
A simple solution is to pass automaticLayout as a prop instead of statically setting it, so I think that's a good solution.

Reply:
@rohinivsenthil  could you checkout this PR?